infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultClusterManager.java

```diff
@@ -81,29 +76,52 @@ public class DefaultClusterManager implements CorfuReplicationClusterManagerAdap
 
     private ConfigStreamListener configStreamListener;
 
+    @Getter
     private String corfuEndpoint = "localhost:9000";
 
     private DefaultClusterConfig topology;
 
+    @Getter
+    public CorfuReplicationDiscoveryService corfuReplicationDiscoveryService;
+
+    @Getter
+    public TopologyDescriptor topologyConfig;
+
+    public String localEndpoint;
+
+    @Setter
+    private String localNodeId;
+
+    private CountDownLatch waitForTopologyInit = new CountDownLatch(1);
```

Comment:
yup, but the implementation of this file entirely is for testing...so not sure about having the annotation for this one field

Reply:
I agree.  But when we read the code to understand the flow from different classes, this class is useful.  For anyone not familiar with how this class is used in tests, it can be confusing to see test-related variables.

Can you add a short comment explaining the role of this latch in tests?  I know some methods were added before this change but adding this info for new variables will be useful :)